week3/homework/src/actions.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+const uuid = require('uuid/v4');
+const { writeFile, readFile } = require('fs');
+const { promisify } = require('util');
+const writeFileWithPromise = promisify(writeFile);
+const readFileWithPromise = promisify(readFile);
+const TODO_PATH = 'todo.json';
+const list = {};
+
+// Read the file with promise 
+function readTodoList() {
+  return readFileWithPromise(TODO_PATH, 'utf8')
+    .then(JSON.parse)
+    .catch((() => ([])));
+}
+
+// Write To do with promise
+function writeTodoList(todo) {
+  return writeFileWithPromise(TODO_PATH, JSON.stringify(todo, null, 2));
+}
+
+// Create todo
+async function createTodo(todo) {
```

Comment:
This is the same code as in Knar's homework #205 down to the mistakes. What's up with that?